roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -4,6 +4,12 @@ host = {{ canonical_hostname }}
 
 [libvirt]
 live_migration_with_native_tls = {{ edpm_nova_live_migration_native_tls|bool }}
+
 {% if edpm_nova_live_migration_tls|bool %}
-live_migration_scheme = tls
+live_migration_uri = qemu+tls://%s/system
```

Comment:
+1 we had talks about replaceing this with the URI during the ptg last week so I'm glad to see this included here